webrtc.html

```diff
@@ -7526,8 +7526,9 @@ <h3>Simulcast functionality</h3>
         attribute to <code>false</code>, or can be reactivated by setting the <code>active</code>
         attribute to <code>true</code>.  Using <code>setParameters</code>, stream characteristics can be
         changed by modifying attributes such as <code>maxBitrate</code> and <code>maxFramerate</code>.</p>
-        <p>As defined in <span data-jsep="simulcast">[[!JSEP]]</span>, support for a several features
-        described in [[MMUSIC-SIMULCAST]] is not required. These include </p>
+        <p>As defined in <span data-jsep="simulcast">[[!JSEP]]</span>, several features described in
+        [[MMUSIC-SIMULCAST]] are not supported. These include support for alternatives and restrictions.
```

Comment:
```suggestion
        [[MMUSIC-SIMULCAST]] are not supported. These include support for simulcast layer alternatives, 
        RID restrictions, RID formats and RID dependencies.
```
The spec distinguishes between some of the different roles restrictions can represent, so i think we should call them out explicitly:
format (pt=) is separate from the restrictions and must come before all of them. 
rid dependencies is only somewhat distinguished, so i'm ok with not specifying that explicitly.